backend/webapi/Features/Parties/ProfileStatus.Model.cs

```diff
@@ -294,7 +294,7 @@ protected override StatusCode Compute(ProfileData profile)
             {
                 return profile switch
                 {
-                    { UserIsBCProvider: false } or { HasPrpAuthorizedLicence: false } => StatusCode.Locked,
+                    { UserIsBCProvider: false } => StatusCode.Locked,
```

Comment:
This should be `HasBCProviderCredential` rather than `UserIsBCProvider`.
The second one indicates the user is currently logged into a BC Provider account.